src/main/java/net/fabricmc/loom/task/RemapJarTask.java

```diff
@@ -121,6 +125,18 @@ public RemapJarTask() {
 		getJarType().set("classes");
 	}
 
+	@InputFiles
+	public FileCollection getNestedDependenciesResolved() { // for task up-to-date checking
+		final ConfigurableFileCollection files = getProject().files();
+
+		if (getAddNestedDependencies().get()) {
+			getNestedConfigurations().get().forEach(files::from);
+			files.from(getNestedJars());
+		}
+
+		return files;
+	}
```

Comment:
How is this any different from putting the `Configuration` in the existing `ConfigurableFileCollection`